plugins/uefi/fu-plugin-uefi.c

```diff
@@ -241,7 +241,7 @@ fu_plugin_uefi_write_splash_data (FuPlugin *plugin,
 
 	/* save to a predicatable filename */
 	esp_path = fu_volume_get_mount_point (data->esp);
-	directory = fu_uefi_get_esp_path_for_os (esp_path);
+	directory = fu_uefi_get_esp_path_for_os (device, esp_path);
```

Comment:
if passing device into this function no need to be passing `esp_path`.  It can be read from device metadata.

Reply:
I may not understand your point. If `esp_path` can be read form metadata, does that mean it is provided by uefi.conf? Or `EnableUefiRemovablePath` should be a string not boolean? Can you please specify more precisely?

Reply:
Actually this one is correct, never mind.